modules/ROOT/pages/alerts-on-runtime-manager.adoc

```diff
@@ -1,317 +1,460 @@
-= Configure Alerts
+= Alerts
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 
-Set up alerts that send an email notification when certain events occur. These can be events for an application or server. Runtime Manager includes several *standard alert* types that trigger notifications for common events, for example when:
 
-* An application exceeds a certain number of events processed within a certain time period.
-* A deployment completes with success or failure.
-* CloudHub encounters a problem with the worker on an application via the worker monitoring system.
-* A server is disconnected or deleted.
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
+image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-CloudHub also enables you to create a custom alert for an application. Custom alerts are triggered by notifications that your application sends to Runtime Manager console using the CloudHub Connector. See xref:custom-application-alerts.adoc[Custom Application Alerts] for more information. Custom alerts are available only for applications deployed to CloudHub. You cannot configure an alert for an application deployed to a customer-hosted runtime.
 
-== Access Alerts
+You can set up alerts, which send email notifications when application or server events occur. 
+
+Runtime Manager includes several standard alert types that trigger notifications for common events:
 
-You can view alerts in Runtime Manager in a few ways.
+* An application exceeds a specific number of events processed within a timeframe.
+* A deployment completes with success or failure.
+* CloudHub encounters a problem with the worker on an application via the worker monitoring system.
```

Comment:
At first I wondered what does "via the worker monitoring system" modify:
```suggestion
* The worker monitoring system notifies CloudHub about a problem with the worker on an application.
```
"the worker" or "a" worker?